From baf0df9fbd23bf033a3b9387548ee0e1bfc1cb8a Mon Sep 17 00:00:00 2001
From: Dan Callaghan <djc@djc.id.au>
Date: Thu, 11 Jun 2015 14:08:44 +1000
Subject: [PATCH 1/2] relax version in kerberos requirement

Any version of the kerberos module should work fine, it has had no API changes in living memory as far as I can see.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 91472e1..42eb4ed 100755
--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@
 if sys.platform == 'win32':
     requires.append('kerberos-sspi')
 else:
-    requires.append('kerberos==1.1.1')
+    requires.append('kerberos')
 
 path = os.path.dirname(__file__)
 desc_fd = os.path.join(path, 'README.rst')
-- 
1.9.3

